### ISSUE TITLE
Another parsing error from Swagger Petstore example

### ISSUE THREAD

**Issue by hooliooo** [mattpolzin/OpenAPIKit#56]
So I downloaded the new 0.29 version and came across a new error:
`- Found neither a $ref nor a Parameter in .parameters[0] for the **GET** endpoint under /store/order/{orderId}`

I used the OpenAPI yaml described here: https://editor.swagger.io/?_ga=2.63340079.1270489904.1587138107-483118911.1586550581#

**Comment by mattpolzin** [comment]
This problem occurred because an `"integer"` minimum was being written as a floating point (albeit a whole-number floating point). As a point of interest, foundations `JSONDecoder` is totally fine with decoding a whole number float as an Int. The Yams `YAMLDecoder`, on the other hand, does not like such things.

This will be fixed with the next release. I have now checked that the rest of the document you linked to successfully parses (it is indeed different than the one I added to the compatibility suite: https://raw.githubusercontent.com/swagger-api/swagger-petstore/master/src/main/resources/openapi.yaml).

Thanks again for submitting these reports!
